### PR TITLE
Adding hover effect over the login now and register now texts

### PR DESCRIPTION
**Pull request by anuukriti**
## Description
Fixed issue  #322 

## Related Issue
[Cite any related issue(s) this pull request addresses. If none, simply state "None"]

## Screenshots / GIFs (if applicable)
[Attach any relevant screenshots or GIFs demonstrating the changes]

## Checklist:
- [ ] I have performed a self-review of my code
- [ ] I have added/updated relevant documentation (if needed)
- [ ] I have tested the changes locally and they function as expected
- [ ] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

File: src/Components/shared/LoginTextLink.jsx [hunk 1]
```diff
@@ -12,12 +12,23 @@ import { Link } from "react-router-dom";
  * @param {string} [props.bottomPosition] - The bottom CSS property for the text link.
  * @returns {JSX.Element} A JSX element representing the LoginTextLink component.
  */
-function LoginTextLink({ text, link, linkText, position, topPosition, bottomPosition ,leftPosition, rightPosition}) {
+function LoginTextLink({
```
Comment (codewarnab) [inline src/Components/shared/LoginTextLink.jsx:15]: make this in one line

File: src/Components/shared/LoginTextLink.jsx
Comment (codewarnab) [inline src/Components/shared/LoginTextLink.jsx:1]: add a screenshot or gif of your work

**Review by codewarnab — CHANGES_REQUESTED** [review]

File: src/Components/shared/LoginTextLink.jsx
Comment (anuukriti) [reply src/Components/shared/LoginTextLink.jsx:1]: Screenshot 2024-05-18 202714

https://github.com/Innovateninjas/Paws-frontend/assets/97687618/77d56dd1-645d-439d-bb54-5eaccbdb7811

Added the hover effect over the login now and register now text
#359

please merge my pull request

File: src/Components/shared/LoginTextLink.jsx (hunk 1 above)
Comment (anuukriti) [reply src/Components/shared/LoginTextLink.jsx]: done, please merge my pull request
Screenshot 2024-05-18 203411

**Review by codewarnab — APPROVED** [review]

**Pull request merged**